examples/fishjam-chat/app.json

```diff
@@ -38,7 +38,8 @@
       ]
     },
     "androidStatusBar": {
-      "hidden": true
+      "backgroundColor": "#BFE7F8",
```

Comment:
**suggestion:** 
use `BrandColors.seaBlue40` from `Colors.ts`

Reply:
That will be difficult, as this is JSON file - and there is no easy way to import consts here :(